src/components/OIO/index.js

```diff
@@ -22,15 +23,36 @@ export default class OIO extends Component {
       OIOStyles: React.PropTypes.object
    }
 
+   constructor(props) {
+      super(props)
+
+      this.state = { windowSize: '' }
+   }
+
    getChildContext() {
       const OIOStyles = {
          fontFamily: this.props.fontFamily,
-         primaryColor: this.props.primaryColor
+         primaryColor: this.props.primaryColor,
+         windowSize: this.state.windowSize
```

Comment:
Hmm you're not supposed to use `context` for values that change. See: https://facebook.github.io/react/docs/context.html#updating-context